lib/nuntiux/mocker.ex

```diff
@@ -36,8 +35,22 @@ defmodule Nuntiux.Mocker do
   @spec delete(process_name) :: ok
         when process_name: Nuntiux.process_name(),
              ok: :ok
-  def delete(_process_name) do
-    :ok
+  def delete(process_name) do
+    process_pid = mocked_process(process_name)
+    reregister(process_name, process_pid)
+  end
+
+  @doc false
+  @spec mocked_process(process_name) :: pid
+        when process_name: Nuntiux.process_name(),
+             pid: pid()
+  def mocked_process(process_name) do
+    {:dictionary, dict} =
```

Comment:
I get where the `:dictionary` is coming from, but I don't really think it adds to the API and, because of it, we can't pipe to the last function call 😢

Reply:
Not using pipes would've solved that problem :P

Reply:
No, I get it, but:

1. I do feel it fits quite nicely here (I don't know if there's a specific idiomatic use case for it, as I've always used it to just "chain operations")
2. the "problem" is really not in the pipeline operator (it just seems to "surface" it): I don't know why the process info fun would return the key too - seems like a weird API choice :)